frontend/src/features/permits/components/permit-list/PermitRowOptions.tsx

```diff
@@ -37,13 +39,14 @@ export const PermitRowOptions = ({
   isExpired: boolean;
   permitId: string;
 }) => {
+  const navigate = useNavigate();
   /**
    * Action handler upon a select event.
    * @param selectedOption The option that was selected.
    */
   const onSelectOptionCallback = (selectedOption: string) => {
     if (selectedOption === PERMIT_ACTION_OPTION_TYPES.VIEW_RECEIPT) {
-      viewReceiptPdf(permitId);
+      viewReceiptPdf(permitId, () => navigate(ERROR_ROUTES.DOCUMENT_UNAVAILABLE));
```

Comment:
Call this ```navigate``` from the parent component, and pass that method as "onDocumentDownloadFail" prop to PermitRowOptions, and call that method here

Reply:
Used onDocumentUnavailable instead of onDocumentDownloadFail for consistency.